xml/obs_notifications.xml

```diff
@@ -300,4 +300,50 @@
    </listitem>
   </itemizedlist>
  </sect1>
+ <sect1 xml:id="cha.obs.notifications.api">
+   <title>API</title>
+   <para>
+     Every user can check their unread notifications by querying:
+   </para>
+   <screen>osc api -X GET "/my/notifications"</screen>
+   <para>
+     This command will return a list with your unread notifications:
+   </para>
+   <screen language="xml">
+&lt;?xml version="1.0" encoding="UTF-8"?&gt;
+&lt;notifications count="2"&gt;
+    &lt;total_pages&gt;1&lt;/total_pages&gt;
+    &lt;current_page&gt;0&lt;/current_page&gt;
+    &lt;notification id="3"&gt;
+        &lt;title&gt;test 1&lt;/title&gt;
+        &lt;who&gt;Administrator&lt;/who&gt;
+        &lt;state&gt;new&lt;/state&gt;
+        &lt;when/&gt;
+        &lt;event_type&gt;review_wanted&lt;/event_type&gt;
+    &lt;/notification&gt;
+    &lt;notification id="25"&gt;
+        &lt;title&gt;test 2&lt;/title&gt;
+        &lt;who&gt;User 2&lt;/who&gt;
+        &lt;event_type&gt;comment_for_package&lt;/event_type&gt;
+        &lt;when/&gt;
+    &lt;/notification&gt;
+&lt;/notifications&gt;
+   </screen>
+   <para>
+     To check the contents of a single notification, query the following endpoint:
+   </para>
+   <screen>osc api -X GET "/my/notifications/3"</screen>
+   <para>
+     The command will return the content of that notification.
+   </para>
+   <screen language="xml">
+&lt;?xml version="1.0" encoding="UTF-8"?&gt;
+&lt;status code="anonymous_user"&gt;
+    &lt;summary&gt;Anonymous user is not allowed here - please login&lt;/summary&gt;
+&lt;/status&gt;
```

Comment:
That's an "Anonymous user" error reponse...

Reply:
Yes, I'm trying to figure out why it is returning this, even when I'm logged in...